zstash/extract.py

```diff
@@ -69,12 +66,13 @@ def multiprocess_extract(num_workers, matches, keep_files, keep_tars, cache):
         workers_work, worker_idx = heapq.heappop(work_to_workers)
         workers_to_tars[worker_idx].add(tar)
         # Add this worker back to the heap, with the new amount of work.
-        heapq.heappush(work_to_workers, (workers_work + tar_to_size[tar], worker_idx))
+        worker_tuple: Tuple[float, int] = (workers_work + tar_to_size[tar], worker_idx)
+        # FIXME: error: Cannot infer type argument 1 of "heappush"
+        heapq.heappush(work_to_workers, worker_tuple)  # type: ignore
```

Comment:
`heapq.heappush` isn't using type annotation, so  it defaults the arguments to `Any`. 

```
def heappush(heap, item):
    """Push item onto heap, maintaining the heap invariant."""
    heap.append(item)
    _siftdown(heap, 0, len(heap)-1)
```
https://github.com/python/cpython/blob/master/Lib/heapq.py#L130

Since you infer the type for `worker_tuple` as not `Any` , mypy throws `Cannot infer type argument 1 of "heappush"mypy(error)` 

For some reason, it only affects the `item` arg and not `heap` even though both args are type `Any`.

Reply:
I think we can just use `# type: ignore` here

Reply:
Isn't the whole point of `Any` that any type can be used though?